docs/ignoring-files-folders-code.md

```diff
@@ -100,18 +94,30 @@ To define files and folders in Semgrep AppSec Platform:
 
 1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login?return_path=/manage/projects).
 2. From the Dashboard Sidebar, select **[Projects](https://semgrep.dev/orgs/-/projects)** > **[Project name]**.
-3. Select the name of the project to modify, and then click the respective <i class="fa-solid fa-gear"></i> **gear** icon in the Settings column.
-4. Enter files and folders to ignore in the **Path Ignores** box.
+3. Find the project you want to modify, then click its <i class="fa-solid fa-gear"></i> **gear** icon in the **Settings** column.
+4. To define files and folders that Semgrep Code and Semgrep Supply Chain ignore:
+   1. Click **Code (SAST) & Supply Chain (SCA)** to expand and display the **Path Ignores** box.
+   2. Enter files and folders to ignore in the **Path Ignores** box.
+   3. Click **Save changes**.
+5. To define files and folders that Semgrep Secrets ignores:
+   1. Click **Secrets** to expand and display the **Path Ignores** box.
+   2. Enter files and folders to ignore in the **Path Ignores** box.
+   3. Click **Save changes**.
 
-Including files and folders through this method is **additive**. When Semgrep AppSec Platform makes a scan, it looks for a `.semgrepignore` within the repository. If no `.semgrepignore` file is found, Semgrep temporarily creates one and adds items from Semgrep AppSec Platform's Path Ignores. Adding items to the **Path Ignores** box does not override default Semgrep ignore patterns.
+Including files and folders through this method is **additive**. When Semgrep AppSec Platform makes a scan, it looks for a `.semgrepignore` within the repository. If no `.semgrepignore` file is found, Semgrep temporarily creates one and adds items from Semgrep AppSec Platform's Path Ignores.
 
-You can also add files to `.semgrepignore` while triaging individual findings in the **No grouping** view on the Findings page:
+Adding items to Semgrep AppSec Platform's **Path Ignores** box doesn't override default Semgrep ignore patterns included with its CLI tool, since the patterns are additive. However, items added to `.semgrepignore` override default Semgrep CLI patterns.
 
-<IgnoreIndividualFindingNoGrouping />
+### Add items to `.semgrepignore` during findings triage
 
-:::note
-Add files to `.semgrepignore` in the fifth step of the procedure described above.
-:::
+You can also add files to `.semgrepignore` while triaging individual findings using Semgrep AppSec Platform in the **No grouping** view on the Findings page:
```

Comment:
This doesn't appear to be possible from the Findings list view anymore. It is possible from either grouping mode once you get to the finding details page (since that's just a page for one finding either way).

I'm not actually sure if this is intended behavior, so it could be worth following up with the team. But for now I would say just change this to "while triaging individual findings using Semgrep AppSec Platform" and then say that you first go to the Details page in Step 2 below.